src/routes/backupTarget/index.js

```diff
@@ -0,0 +1,232 @@
+    }
+
+    const backupTargetBulkActionsProps = {
+      selectedRows,
+      bulkDeleteBackupTargets(records) {
+        dispatch({
+          type: 'backupTarget/bulkDelete',
+          payload: records,
+        })
+      },
+    }
+
+    const backupTargetFilterProps = {
+      location,
+      defaultField: 'name',
+      fieldOption: [
+        { value: 'name', name: 'Name' },
+        { value: 'backupTargetURL', name: 'URL' },
+        { value: 'credentialSecret', name: 'Credential Secret' },
+        { value: 'pollInterval', name: 'Poll Interval' },
+        { value: 'available', name: 'Available' },
+      ],
+      booleanFields: ['available'],
+      onSearch(filter) {
+        const { field: filterField, value: filterValue, booleanValue: filterBooleanValue } = filter
+        if (filterField && (filterValue || typeof filterBooleanValue === 'boolean')) {
+          const queryStringObj = { field: filterField }
+          if (['available'].includes(filterField) && typeof filterBooleanValue === 'boolean') {
+            queryStringObj.booleanValue = filterBooleanValue
+          }
+          if (['name', 'backupTargetURL', 'credentialSecret', 'pollInterval'].includes(filterField) && filterValue) {
+            queryStringObj.value = filterValue
+          }
+          dispatch(routerRedux.push({
+            pathname: '/backupTarget',
+            search: queryString.stringify(queryStringObj),
+          }))
+        } else {
+          dispatch(routerRedux.push({
+            pathname: '/backupTarget',
+            search: queryString.stringify({}),
+          }))
+        }
+      },
+    }
+
+    return (
+      <div className="content-inner" style={{ display: 'flex', flexDirection: 'column', overflow: 'visible !important' }}>
+        <Row gutter={24} style={{ marginBottom: 8 }}>
+          <Col lg={{ span: 4 }} md={{ span: 6 }} sm={24} xs={24}>
+            <BackupTargetBulkActions {...backupTargetBulkActionsProps} />
+          </Col>
+          <Col lg={{ offset: 13, span: 7 }} md={{ offset: 8, span: 10 }} sm={24} xs={24}>
+            <Filter {...backupTargetFilterProps} />
+          </Col>
+        </Row>
+        <Button className="out-container-button" size="large" type="primary" disabled={loading} onClick={() => this.handleCreateModalOpen()}>
+          Create Backup Target
+        </Button>
+        <BackupTargetList {...backupTargetListProps} />
+        {createBackupTargetModalVisible && <CreateBackupTargetModal {...createBackupTargetModalProps} />}
+        {editBackupTargetModalVisible && <EditBackupTargetModal {...editBackupTargetModalProps} />}
+      </div>
+    )
+  }
+}
+
+BackupTarget.propTypes = {
+  app: PropTypes.object,
+  backupTarget: PropTypes.object,
+  loading: PropTypes.bool,
+  location: PropTypes.object,
+  dispatch: PropTypes.func,
+}
+
+export default connect(({ app, backupTarget, loading }) => ({ app, backupTarget, loading: loading.models.backupTarget }))(BackupTarget)
```

Comment:
[Question] 
Do we want the user to set the default backup target in `Settings -> General` or `Settings -> Backup Target`?
If `Backup Target` is the case, maybe the section in `General` can be removed to reduce confusion.

Reply:
Same question from me.

https://github.com/longhorn/longhorn-ui/pull/755#issuecomment-2547857355

Reply:
Just talked to @mantissahz , he will have next PR to align the default backup target in setting and backup target page.